### DISCUSSION TITLE
Etsy Product Fetch Issue

### DISCUSSION THREAD

**Discussion by saluja-anchanto** [etsy/open-api#1266] [Q&A]
Hi Team,

When we are fetching products from etsy . We are facing error "{\"error\":\Both products must have the same number of variations\"}".
Url: https://openapi.etsy.com/v3/application/listings/1058101490/inventory?includes=listing&show_deleted=true
GET Request

Can you please tell use why we are receiving this error?

**Comment by saluja-anchanto** [comment]
Hi Team,

Can anyone help regarding this issue?

**Reply by turboturtleD** [reply]
Does it do the same thing if you capitalize ‘Listing’ in your includes?

https://openapi.etsy.com/v3/application/listings/1058101490/inventory?includes=Listing&show_deleted=true

**Comment by saluja-anchanto** [comment]
Hi @turboturtleD ,

We have tried the suggested changes . Still we are receiving same 
Url: https://openapi.etsy.com/v3/application/listings/1058101490/inventory?includes=Listing&show_deleted=true
{
    "error": "Both products must have the same number of variations"
}
<img width="976" alt="Screenshot 2024-07-26 at 12 27 02 PM" src="https://github.com/user-attachments/assets/34ce9b72-f8ee-4762-8f73-d37f415e588c">

**Comment by saluja-anchanto** [comment]
Hi @turboturtleD  ,

Any update on this?

**Comment by saluja-anchanto** [comment]
Hi @turboturtleD 

Any update on this?

**Comment by turboturtleD** [comment]
Hey saluja, I'm not an admin.  Sorry, I don't have any answers for you.  I assume maybe you have some bad data saved in your Etsy listing inventory data that is causing this error.  Do you have the same error with other listings?

**Reply by upadhyaydheeraj** [reply]
yes we are getting same error for other listing also

**Reply by upadhyaydheeraj** [reply]
Team Please help here

**Reply by turboturtleD** [reply]
Since this is just a GET request, and you aren't getting an authentication error, it's hard to not suspect that you have a malformed listing(s) data.  Try removing all of the variations using the Etsy front end, then attempt a GET request.  If it works, try adding the variations one at a time and see what happens.
